android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java

```diff
@@ -201,11 +202,15 @@ public Integer getTotalDiskCapacity() {
   }
 
   @ReactMethod
-  public Integer getFreeDiskStorage() {
+  public Long getFreeDiskStorage() {
     try {
-      StatFs external = new StatFs(Environment.getExternalStorageDirectory().getAbsolutePath());
-      return external.getAvailableBlocks() * external.getBlockSize();
-    } catch (Exception e) {
+      File path = Environment.getExternalStorageDirectory();
+      SatFs statFs = new StatFs(path.getPath());
```

Comment:
Why did we switch from `getAbsolutePath` to `getPath` here?

Reply:
The PR I copied this from switched. Just did quick search, seems like people use `getAbsolutePath`.